packages/core/src/sass/app/_common/components/onfido-container.scss

```diff
@@ -258,6 +258,15 @@
             }
         }
     }
+
+    .additional-field {
+        @include desktop {
```

Comment:
do we need to explicitly mention `desktop`? by default we can set it to 1.6rem right?